services/harmony/test/service-image-tags.ts

```diff
@@ -894,10 +1092,93 @@ describe('Service image endpoint', async function () {
           it('returns the tag we sent', async function () {
             expect(this.res.body.tag).to.eql('foo');
           });
+
+          it('returns statusLink', async function () {
+            const link = this.res.body.statusLink;
+            statusPath = new URL(link).pathname;
+            expect(link).to.include('http://127.0.0.1:4000/service-image-tag/deployment/');
+            linkDeploymentId = getDeploymentIdFromStatusLink(link);
+            expect(linkDeploymentId).to.not.be.null;
+          });
+
+          it('reaches a terminal status without timeout', async function () {
```

Comment:
Does this really test anything, given your stubs?

Reply:
This essentially performs a wait to make sure the callback has completed. So it is necessary to make sure this test does not interfere with the following tests.